setup.py

```diff
@@ -17,9 +17,9 @@ def prepare_zlib():
     global include_dirs
     global sources
 
-    zlib_dir = os.path.join(root_dir, "zlib-1.2.13")
-    zlib_file = os.path.join(root_dir, "zlib-1.2.13.zip")
-    url = "https://github.com/madler/zlib/releases/download/v1.2.13/zlib1213.zip"
+    zlib_dir = os.path.join(root_dir, "zlib-1.3")
+    zlib_file = os.path.join(root_dir, "zlib-1.3.zip")
+    url = "https://zlib.net/zlib13.zip"
```

Comment:
@aroon-color - can/should we just make a separate PR for this?

Reply:
Good point, we should probably have a set of stacked PRs since we pin the commit associated with this PR